pkg/interop/client.go

```diff
@@ -60,9 +60,10 @@ type jsRPCPaths struct {
 }
 
 func (p jsRPCPaths) join() string    { return p.Join }
-func (p jsRPCPaths) rejoin() string  { return p.Rejoin }
 func (p jsRPCPaths) appSKey() string { return p.AppSKey }
-func (p jsRPCPaths) homeNS() string  { return p.HomeNS }
+
+//func (p jsRPCPaths) rejoin() string  { return p.Rejoin }
```

Comment:
Can be removed ?

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/pull/4917/commits/3f51ddc999b8cf21b8580b65ebecf1112b9b5808